docs/application/vscode-ext/web.md

```diff
@@ -223,88 +204,82 @@ To debug application, follow these steps:
 > **Note**
 >
 > Ensure the following:
-> - Emulator is launched
-> - Device is connected to the PC
+> - Emulator is launched, or device is connected to the PC
 > - Chrome browser is installed
 
-1. Open **Command Palette** and enter **Debug Tizen App**.
-2. Enter the name of the Tizen Web App you want to run.
-3. A successful popup is created and the app is launched.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Debug Tizen App** and press **Enter**.
+3. Enter the name of the Tizen Web App you want to run. 
 
-> **Note**
-> 
-> In addition, `vscode` terminal displays successful launch log messages:
->
-> - debug `0` means normal launch.
-> - debug `1` means launching an inspector available from command line
-> - additionally, debug the connected port as shown in the following log
+    A successful popup is created and the app is launched. Also, you will see a successful launch log in the terminal on the vscode.
```

Comment:
Suggestion: Can we replace "A successful popup is created" with

A notification appears and the app...

Please verify.

Reply:
done.